src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+// / - `<Todo />` is a component that takes in the`todo` data and displays the task to the screen.
+
+const Todo = props => {
+	return props.list.map((item, id) => (
+		<li
+			onClick={() => props.toggleComplete(item.id)}
+			key={id}
+			id={id}
+			style={{
+				textDecoration: item.completed ? 'line-through' : 'none'
```

Comment:
Nice job using inline styles here. Some argue that it's not best practice, but in this case, when you need to apply a style based on some simple logic, this is a very direct route.